WTWSMS/history/characters/scottish.txt

```diff
@@ -52,7 +52,7 @@
 	intrigue=9
 	stewardship=9
 	religion="nicene"
-	culture="scottish"
+	culture=levantine_arabic
```

Comment:
Arabic? Isn't that anachronistic here?

Reply:
This is what some other Apostles use. St. Andrew was born in a Jewish family though. What culture represents the Jews from Galilee?

Reply:
Hebrew maybe?

Reply:
Hebrew probably.

Reply:
Done, changed to hebrew